osquery/remote/serializers/json.cpp

```diff
@@ -21,7 +21,8 @@ Status JSONSerializer::serialize(const pt::ptree& params,
   try {
     pt::write_json(output, params, false);
   } catch (const pt::json_parser::json_parser_error& e) {
-    return Status(1, std::string("JSON serialize error: ") + e.what());
+    LOG(WARNING) << "Error serializing JSON: " << e.what();
```

Comment:
I think you might be right, as the only thing which can be provided for context within the serialize/deserialize functions are the string/pt::tree being parsed, which would probably not provide the necessary context (and clog up the logs as well).